beesdoo_shift_swap/models/res_partner.py

```diff
@@ -226,9 +226,7 @@ def exchange_shifts(
             if not new_shift_generated:
                 new_shift = wanted_tmpl_dated.new_shift(self)
                 if solidarity_offer:
-                    new_shift.write(
-                        {"solidarity_offer_ids": [(6, 0, solidarity_offer.ids)]}
-                    )
+                    new_shift.solidarity_offer_ids = [(6, 0, solidarity_offer.ids)]
```

Comment:
What is the reason for this change?

Reply:
It didn't work with write(), don't know why